ios/FluentUI.Demo/FluentUI.Demo/Demos/CardNudgeDemoController.swift

```diff
@@ -260,6 +260,15 @@ extension CardNudgeDemoController: DemoAppearanceDelegate {
     // MARK: - Custom tokens
 
     private class ThemeWideOverrideCardNudgeTokens: CardNudgeTokens {
+        override var outlineWidth: CGFloat {
+            return 10.0
+        }
+
+        override var outlineColor: DynamicColor {
+            return DynamicColor(light: GlobalTokens().sharedColors[.darkRed][.tint50],
```

Comment:
Can you store the GlobalTokens in a constant instead of instantiating it twice? 
I don't think it'll be a big deal in this case, but it's a good way to set the example for the client apps on how to retrieve these tokens.